1-js/05-data-types/02-number/article.md

```diff
@@ -423,12 +423,12 @@ For different numeral systems:
 
 For converting values like `12pt` and `100px` to a number:
 
-- Use `parseInt/parseFloat` for the "soft" conversion, which reads a number from a string until it can. 
+- Use `parseInt/parseFloat` for the "soft" conversion, which reads a number from a string until broken by something that is not a number. 
```

Comment:
"...and then return the value they could read before the error?"

Reply:
see my comments above

Reply:
So: 'Use `parseInt/parseFloat` for the "soft" conversion, which reads a number from a string and then returns the value they could read before the error.'